.travis.yml

```diff
@@ -3,6 +3,7 @@ language: node_js
 node_js:
   - v12.10.0
 script:
+  - commitlint-travis
```

Comment:
Taken right from [the CI setup guide](https://commitlint.js.org/#/guides-ci-setup).